pkg/plugins/platform/local.go

```diff
@@ -258,6 +258,15 @@ func (l *LocalApp) Start() error {
 	composePath := l.DockerComposeYAMLPath()
 	l.DockerEnv()
 
+	existing, err := l.FindContainerByType("web")
+	if err != nil && !strings.Contains(err.Error(), "could not find containers which matched") {
+		return err
+	}
+
+	if existing.State == "running" {
```

Comment:
There is a `SiteStatus()` method coming in the list rework. It would probably be wise to use that abstraction once it lands over accessing state directly.